MCprep_addon/mcprep_ui.py

```diff
@@ -21,6 +21,9 @@
 import bpy
 import os
 
+from bpy.types import DATA_PT_EEVEE_shadow, WM_MT_splash_quick_setup
+from bpy.utils import register_class
```

Comment:
Also unused (all references below are using `bpy.utils.register_class`)

Reply:
Ahh, didn't notice that. I think my IDE automatically added those in for some reason or another, I'll go ahead and remove that